deployments/compose/dbt/dbt-project/.gitignore

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
remove empty line

Reply:
Generated as part of the `dbt init` command, not changing

Reply:
strange - I guess the developer didn't want dbt list to be just right after the current gitignore list - so they added empty line first :)